### PR TITLE
changes to incorporate Proof of Transfer

### PR DESCRIPTION
**Pull request by joshmatix**
**Added new files file:**
pages/pox/overview.md
public/images/pox.png

**Changes to existing files:**

next.config.js
async function redirects() {
  return [
    {
      source: '/pox/overview.html',
      destination: '/pox/overview',
      permanent: true,
    },
	

navigation.yaml
- title: Technology
    pages:
      - path: /pox
        pages:
          - path: /overview
		  
		  
		  
pages/index.md
[@page-reference | grid-small]
| /ecosystem/overview, /ecosystem/stacks-token, /stacks-blockchain/running-testnet-node, /pox/overview

### CODE REVIEW

**Review by agraebe — CHANGES_REQUESTED** [review]
thanks for the PR @joshmatix 🙏 I left some comments, most importantly the nav structure request

File: src/common/navigation.yaml
Comment (agraebe) [inline src/common/navigation.yaml:20]: let's remove this comment, assuming it's a leftover from before. with that, we should be good to go

File: src/pages/index.md
Comment (agraebe) [inline src/pages/index.md:21]: shouldn't `/pox/overview` > `/stacks-blockchain/pox`?

**Review by agraebe — APPROVED** [review]

**Comment by agraebe** [comment]
I resolved a conflict and previewed the changes in gitpod. A few suggestions:

- for the image inside the markdown, we should prefer the markdown style for images: `<alt>`
- leaving the file named as `pox.md` will create `Pox` navigation item, which we should avoid. I'd suggest we rename the file to `proof-of-transfer.md` and update the references
- the markdown file should end with an empty new line. I'd run a linter and fix to ensure we account for every linting rule
- @jasperjansz - I'm marking this for you to review. I think this illustration needs to be updated (can happen after we merge this PR)

**Review by agraebe — APPROVED** [review]

**Comment by joshmatix** [comment]
Since it is approved, do I have to do anything else to this PR?

**Comment by agraebe** [comment]
thanks @joshmatix, we can merge it now but I seem not to be able to trigger that action.

@aulneau - do you have the same issue? merging is disabled due to lack of check completion?

**Comment by agraebe** [comment]
@joshmatix it turns out I can't merge this PR because of a conflict. could you resolve/rebase so we can merge this asap? thanks!

**Comment by joshmatix** [comment]
Will rebasing fix the following error:

Run aulneau/vercel-action@v20.0.0
/home/runner/work/_actions/aulneau/vercel-action/v20.0.0/dist/index.js:6801
throw new Error(`Input required and not supplied: ${name}`);
^
Error: Input required and not supplied: vercel-token
at Object.getInput
(/home/runner/work/_actions/aulneau/vercel-action/v20.0.0/dist/index.js:6801:15)

at Object.104
(/home/runner/work/_actions/aulneau/vercel-action/v20.0.0/dist/index.js:1052:26)

at __webpack_require__
(/home/runner/work/_actions/aulneau/vercel-action/v20.0.0/dist/index.js:22:30)

at startup
(/home/runner/work/_actions/aulneau/vercel-action/v20.0.0/dist/index.js:37:19)

at
/home/runner/work/_actions/aulneau/vercel-action/v20.0.0/dist/index.js:41:18

at Object.<anonymous>
(/home/runner/work/_actions/aulneau/vercel-action/v20.0.0/dist/index.js:44:10)

at Module._compile (internal/modules/cjs/loader.js:959:30)
at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)
at Module.load (internal/modules/cjs/loader.js:815:32)
at Function.Module._load (internal/modules/cjs/loader.js:727:14)

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by aulneau** [comment]
The vercel action will never work in this PR due to it's origin is a fork. If you rebase from `master`, we can test out the deployment via the official vercel github addon.

**Review by joshmatix — COMMENTED** [review]
added to vercel

**Comment by joshmatix** [comment]
I followed the instructions under
https://docs.blockstack.org/ecosystem/contributing#working-locally

Do those instructions have to be updated?



<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by agraebe** [comment]
@joshmatix I'll merge this PR as part of https://github.com/blockstack/docs/pull/897

thanks for the contribution!

**Pull request closed without merging**
